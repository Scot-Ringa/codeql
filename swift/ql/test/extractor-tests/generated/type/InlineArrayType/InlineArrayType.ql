// generated by codegen/codegen.py, do not edit
import codeql.swift.elements
import TestUtils

query predicate instances(
  InlineArrayType x, string getName__label, string getName, string getCanonicalType__label,
  Type getCanonicalType, string getCountType__label, Type getCountType,
  string getElementType__label, Type getElementType
) {
  toBeTested(x) and
  not x.isUnknown() and
  getName__label = "getName:" and
  getName = x.getName() and
  getCanonicalType__label = "getCanonicalType:" and
  getCanonicalType = x.getCanonicalType() and
  getCountType__label = "getCountType:" and
  getCountType = x.getCountType() and
  getElementType__label = "getElementType:" and
  getElementType = x.getElementType()
}
