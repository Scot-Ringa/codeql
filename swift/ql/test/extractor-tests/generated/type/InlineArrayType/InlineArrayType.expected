| [42 of Int] | getName: | [42 of Int] | getCanonicalType: | InlineArray<42, Int> | getCountType: | 42 | getElementType: | Int |
